web/src/features/hfiCalculator/components/MeanIntensityGroupRollup.tsx

```diff
@@ -0,0 +1,132 @@
+import { TableCell, Tooltip } from '@material-ui/core'
+import ErrorOutlineIcon from '@material-ui/icons/ErrorOutline'
+import { createMuiTheme, makeStyles, ThemeProvider } from '@material-ui/core/styles'
+import React from 'react'
+import { isUndefined } from 'lodash'
+import { PlanningArea } from 'api/hfiCalcAPI'
+import { isValidGrassCure } from 'features/hfiCalculator/validation'
+import { StationDaily } from 'api/hfiCalculatorAPI'
+import {
+  calculateMeanIntensityGroup,
+  intensityGroupColours
+} from 'features/hfiCalculator/components/meanIntensity'
+
+export interface MeanIntensityGroupRollupProps {
+  area: PlanningArea
+  dailiesMap: Map<number, StationDaily>
+}
+
+const useStyles = makeStyles({
+  intensityGroupSolid1: {
+    background: intensityGroupColours.lightGreen,
+    fontWeight: 'bold',
+    textAlign: 'center'
+  },
+  intensityGroupSolid2: {
+    background: intensityGroupColours.cyan,
+    fontWeight: 'bold',
+    textAlign: 'center'
+  },
+  intensityGroupSolid3: {
+    background: intensityGroupColours.yellow,
+    fontWeight: 'bold',
+    textAlign: 'center'
+  },
+  intensityGroupSolid4: {
+    background: intensityGroupColours.orange,
+    fontWeight: 'bold',
+    textAlign: 'center'
+  },
+  intensityGroupSolid5: {
+    background: intensityGroupColours.red,
+    fontWeight: 'bold',
+    color: 'white',
+    textAlign: 'center'
+  },
+  alignErrorIcon: {
+    marginTop: '6px',
+    textAlign: 'center'
+  }
+})
+
+const errorIconTheme = createMuiTheme({
+  overrides: {
+    MuiSvgIcon: {
+      root: {
+        fill: '#D8292F'
+      }
+    }
+  }
+})
+
+const toolTipFirstLine = 'Grass Cure % not defined in WFWX for one or more stations.'
+const toolTipSecondLine = 'Cannot calculate Mean FIG.'
+const toolTipElement = (
+  <div>
+    {toolTipFirstLine}
+    {toolTipSecondLine}
```

Comment:
Won't this need a `<br />`?

Reply:
It looks worse without, it was a `<br />` a couple of commits up.

Reply:
https://github.com/bcgov/wps/pull/1280/commits/3e61ddbb8be3ba80a2e1845b0940ac3c8bbbba61